politeiad/backendv2/tstorebe/plugins/pi/cmds.go

```diff
@@ -165,6 +169,192 @@ func tokenMatches(cmdToken []byte, payloadToken string) error {
 	return nil
 }
 
+// cmdSummary returns the pi summary of a proposal.
+func (p *piPlugin) cmdSummary(token []byte) (string, error) {
+	// Get record metadata
+	r, err := p.recordAbridged(token)
+	if err != nil {
+		return "", err
+	}
+	mdState := r.RecordMetadata.State
```

Comment:
Prefer variable lists like this to be inside var brackets so that go fmt aligns them nicely.

```
	var (
		mdState    = r.RecordMetadata.State
		mdStatus   = r.RecordMetadata.Status
		voteStatus = ticketvote.VoteStatusInvalid

		bsc *pi.BillingStatusChange
	)

```

Reply:
oops!